### PR TITLE
Header & footer Redesign implementation

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@sewagodimo @codiebeulaine @saeedmb8 a review please 

Implement basic header & footer
screen shot 2018-06-20 at 14 30 10
screen shot 2018-06-20 at 14 29 54

File: gem/templates/rwanda/core/language_block.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {% if languages|length > 1 %}
-<div class="footer-languages footer-languages{{ extra_classes }}">
-  {% trans "Language" %}:
+<div class="languages languages--footer">
+  <p class="languages__label">{% trans "Language" %}:<p>
   {% get_current_language as LANGUAGE_CODE %}
   {% for language in languages %}
-    <a href="{% url 'locale_set' language.locale %}?next={{request.path}}{% if request.GET.urlencode %}?{{ request.GET.urlencode }}{% endif %}" class="languages__item {% if LANGUAGE_CODE == language.locale %} languages__item--active{% endif %}">{{ language }}</a>
+    <a href="{% url 'locale_set' language.locale %}?next={{request.path}}{% if request.GET.urlencode %}?{{ request.GET.urlencode }}{% endif %}" class="languages__anchor {% if LANGUAGE_LOCALE == language.locale %}active{% endif %}">{{ language }}</a>
```
Comment (saeedmb8) [inline gem/templates/rwanda/core/language_block.html]: @Mitso I see you have changed the `LANGUAGE_CODE`  to `LANGUAGE_LOCALE ` but the variable in the tag is still called `LANGUAGE_CODE ` `{% get_current_language as LANGUAGE_CODE %}`

File: gem/templates/rwanda/core/tags/latest_listing_homepage.html [hunk 2]
```diff
@@ -1,5 +1,86 @@
+{#{% if articles %}#}
+{#{% with articles|first as article %}#}
+{#{% include "core/article_intro_partial.html" with article=article show_section_image=True %}#}
+{#{% endwith %}#}
+{#{% endif %}#}
```
Comment (saeedmb8) [inline gem/templates/rwanda/core/tags/latest_listing_homepage.html]: Should we remove these if we don't need them ?

File: gem/templates/rwanda/core/language_block.html (hunk 1 above)
Comment (Mitso) [reply gem/templates/rwanda/core/language_block.html]: @saeedmb8 great spotting that. thank you updating

**Pull request merged**